src/OrchardCore.Build/Dependencies.props

```diff
@@ -15,8 +15,8 @@
     <XunitVersion>2.3.0-beta2-build3683</XunitVersion>
     <XunitRunnerVisualStudioVersion>2.3.0-beta2-build1317</XunitRunnerVisualStudioVersion>
     <FluidCoreVersion>1.0.0-beta-9422</FluidCoreVersion>
-    <YesSqlVersion>2.0.0-beta-1174</YesSqlVersion>
-    <YesSqlProvidersVersion>1.0.0-beta-1174</YesSqlProvidersVersion>
+    <YesSqlVersion>2.0.0-beta-1182</YesSqlVersion>
```

Comment:
FYI this change was already done in `dev`: https://github.com/OrchardCMS/OrchardCore/blob/dev/src/OrchardCore.Build/Dependencies.props#L18-L19

Reply:
I saw, it's just for the Jint one, and the merge will handle it

Reply:
Too lazy to do a rebase on your branch? :sweat_smile: